### ISSUE TITLE
api-Job-Single API s3 path 

### ISSUE THREAD

**Issue by ilissablech** [openaddresses/openaddresses#5898]
Previously, there used to be an s3 path value in the https://batch.openaddresses.io/docs/#api-Job-Single API return. Please correct me if I've confused myself.

For example, this is an old return for `https://batch.openaddresses.io/api/job/147145`. I'm no longer seeing the s3 path. The format of the path seems to be `s3://v2.openaddresses.io/batch-prod/job/<soruce_id>/source.geojson.gz`. But I'd like to confirm:
a) that we are to no longer expect the s3 path to be populated in the api job object
b) the formatting of the s3 path will stay consistent in the `s3://v2.openaddresses.io/batch-prod/job/<soruce_id>/source.geojson.gz` format. 

Thank you!

**Old job return:**
```json
 {
 	"id": 147145,
 	"size": 1621051,
 	"s3": "s3://v2.openaddresses.io/batch-prod/job/147145/source.geojson.gz",
 	"run": 1160,
 	"license": "False",
 	"map": 3457,
 	"created": "2021-08-27T12:06:37.663Z",
 	"source_name": "us/al/st_clair",
 	"source": "https://raw.githubusercontent.com/openaddresses/openaddresses/131b5ea1510fe780389c2f20bfe4b25e7f212db4/sources/us/al/st_clair.json",
 	"layer": "addresses",
 	"name": "county",
 	"output": {
 		"cache": "False",
 		"output": "True",
 		"preview": "True"
 	},
 	"loglink": "batch-prod-job/default/b2190fb3bf2c48549fc83afd9802b3b3",
 	"status": "Success",
 	"version": "1.0.1",
 	"count": 57921,
 	"bounds": {
 		"type": "Polygon",
 		"coordinates": [
 			[
 				[-86.5774677, 33.3959265],
 				[-86.0441925, 33.3959265],
 				[-86.0441925, 33.9879355],
 				[-86.5774677, 33.9879355],
 				[-86.5774677, 33.3959265]
 			]
 		]
 	},
 	"stats": {
 		"counts": {
 			"city": 0,
 			"unit": 0,
 			"number": 55878,
 			"region": 0,
 			"street": 40017,
 			"district": 0,
 			"postcode": 0
 		},
 		"validity": {
 			"valid": 40015,
 			"failures": {
 				"number": 2043,
 				"street": 17904,
 				"geometry": 0
 			}
 		}
 	}
 }
```

**Current job return:**
```json
{
	"id": 147145,
	"size": 1621051,
	"run": 1160,
	"license": false,
	"map": 3457,
	"created": 1630065997663,
	"source_name": "us/al/st_clair",
	"source": "https://raw.githubusercontent.com/openaddresses/openaddresses/131b5ea1510fe780389c2f20bfe4b25e7f212db4/sources/us/al/st_clair.json",
	"layer": "addresses",
	"name": "county",
	"output": {
		"cache": false,
		"output": true,
		"preview": true
	},
	"loglink": "batch-prod-job/default/b2190fb3bf2c48549fc83afd9802b3b3",
	"status": "Success",
	"version": "1.0.1",
	"count": 57921,
	"bounds": {
		"type": "Polygon",
		"coordinates": [
			[
				[
					-86.5774677,
					33.3959265
				],
				[
					-86.0441925,
					33.3959265
				],
				[
					-86.0441925,
					33.9879355
				],
				[
					-86.5774677,
					33.9879355
				],
				[
					-86.5774677,
					33.3959265
				]
			]
		]
	},
	"stats": {
		"counts": {
			"city": 0,
			"unit": 0,
			"number": 55878,
			"region": 0,
			"street": 40017,
			"district": 0,
			"postcode": 0
		},
		"validity": {
			"valid": 40015,
			"failures": {
				"number": 2043,
				"street": 17904,
				"geometry": 0
			}
		}
	}
}```

**Comment by ingalls** [comment]
@ilissablech Requests made by sponsors should show the S3 path. Are you using an API Token to make the requests? All OA accounts with an `@mapbox.com` email should be marked as `sponsors` - which would allow you to view the `s3` job URL. We just cleared our cache to push some larger authentication changes out which may have resulted in the API requests hitting the server directly. A non-authenticated user would not see the S3 link unless it was returned via the cache.

The path itself is not guaranteed to follow any consistent format (bucket/key format may change) so I would continue to load it dynamically from the API but the _contents_ of the file are always, and will continue to be, line-delimited GeoJSON.

So Tl;Dr 
a) Authed user should see the S3 key
b) Don't depend on internal bucket location - this is subject to change

**Comment by ilissablech** [comment]
Thanks @ingalls -- got what I needed to access the S3 path. 🎉
